formats/hocon/build.gradle

```diff
@@ -1,26 +1,12 @@
 /*
- *  Copyright 2018 JetBrains s.r.o.
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
+ * Copyright 2017-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
 apply plugin: 'kotlin'
 apply plugin: 'kotlinx-serialization'
 
-if (rootProject.ext.jvm_ir_enabled) {
-    kotlin.target.compilations.all {
-        kotlinOptions.useIR = true
-    }
+compileKotlin {
+    kotlinOptions.allWarningsAsErrors = true
```

Comment:
Does it affect all source-sets or only `main` one?

Reply:
Since this is `compileKotlin`, not `compileTestKotlin`, I believe only the main one. I can re-check this if you'd like